specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-01-01/examples/ImpactedResources_ListBySubscriptionId.json

```diff
@@ -0,0 +1,44 @@
+﻿{
+	"parameters": {
+		"subscriptionId": "subscriptionId",
+		"api-version": "2018-01-01",
+		"$filter": "service eq 'Virtual Machines' or region eq 'West US'"
+	},
+	"responses": {
+		"200": {
+			"body": {
+				"value": [
+					{
+						"id": "<resourceId>/providers/Microsoft.ResourceHealth/impactedResources/current",
+						"name": "current",
+						"type": "Microsoft.ResourceHealth/impactedResources",
+						"location": "eastus",
```

Comment:
having `location` here indicates that what you return is actually a tracked resource. Please make sure or adjust the example accordingly